protos/telemetry/telemetry.proto

```diff
@@ -576,7 +576,11 @@ enum FixType {
 // Battery type.
 message Battery {
     uint32 id = 3 [(mavsdk.options.default_value)="0"]; // Battery ID, for systems with multiple batteries
+    float temperature_cdegC = 7 [(mavsdk.options.default_value)="NaN"]; // Temperature of the battery. INT16_MAX for unknown temperature
     float voltage_v = 1 [(mavsdk.options.default_value)="NaN"]; // Voltage in volts
+    float current_battery_cA = 4 [(mavsdk.options.default_value)="NaN"]; // Battery current, -1: autopilot does not measure the current
```

Comment:
Wouldn't `A` or `mA` be a bit more natural? :sweat_smile:

Reply:
agree as well. I took the format from the BATTERY_STATUS message but probably not the most meaningful units